google-cloud-compute-v1/samples/default_values.rb

```diff
@@ -47,9 +47,9 @@ def set_usage_export_bucket project:, bucket_name:, report_name_prefix: ""
          "to have the default prefix of `usage_gce`."
   end
   projects_client = ::Google::Cloud::Compute::V1::Projects::Rest::Client.new
-  operation = projects_client.set_usage_export_bucket project: project,
-                                                      usage_export_location_resource: export_location
-  wait_until_done project: project, operation: operation
+  gapic_op = projects_client.set_usage_export_bucket project: project,
```

Comment:
I'm not sure if we want to highlight the name gapic as in `gapic_op`.  I don't think the term appears anywhere in the documentation and developers might wonder where it comes from. How about just keeping the variable name as operation? `operation.operation` sounds a bit weird but this kind of repetition appears here and there.